core/posts/serializers.py

```diff
@@ -20,10 +20,20 @@ def to_representation(self, instance):
         instance.author.host = get_host_url()
         instance.author.url = get_author_url(str(instance.author.id))
         representation['author'] = AuthorSummarySerializer(instance.author, read_only=True).data
+        # From https://docs.djangoproject.com/en/dev/topics/db/queries/#limiting-querysets
+        representation['comments'] = CommentsSerializer(instance.comments.all()[:5], many=True, read_only=True).data
```

Comment:
This should be the last 5 comments
https://github.com/abramhindle/CMPUT404-project-socialdistribution/blob/master/example-article.json#L75

Can you also implement the following keys from the example-article?
count, size, next